src/features/create-dao/steps/DAOSummary.module.scss

```diff
@@ -0,0 +1,70 @@
+.Row {
+	display: flex;
+	margin-top: 0.625rem;
+	width: 20.5rem;
+
+	&_Summary {
+		display: flex;
+		margin-top: 0;
+	}
+
+	.Column {
+		display: flex;
+		flex-direction: column;
+		
+		.Heading {
+			margin: 0 0 1rem 0;
+		}
+		
+		.MediaInput {
+			height: 12.5rem;
+			width: 12.5rem;
+			margin-right: 2rem;
+			border-radius: 0.5rem !important;
+		}
+		
+		.Summary {
+			margin-right: 1.3rem;
+			width: 8rem;
+			height: 2.5rem;
+			margin-bottom: 1rem;
+		}
+
+		&_Second {
+			line-height: 1.5rem;
+		}
+	}
+}
+
+.Text {
+	font-family: 'Inter';
```

Comment:
Shouldn't need to set font-family, font-style, font-weight. This look like the defaults.

Reply:
Good spotting. Have removed.